src/samples/GroupChannelScheduledMessages.js

```diff
@@ -287,7 +370,43 @@ const BasicGroupChannelSample = (props) => {
             return onError(error);
         }
 
-        updateState({ ...state, channels: channels, loading: false, settingUpUser: false });
+        const scheduledMessages = await loadSchedulesMessages()
+
+        updateState({ ...state, channels: channels, loading: false, settingUpUser: false, scheduledMessages });
+    }
+
+    const toggleShowScheduledMessageSettingsModal = () => {
+        updateState({ ...state, showScheduledMessageSettingsModal: !state.showScheduledMessageSettingsModal });
+    }
+
+    const toggleShowScheduledMessageListModal = async () => {
+        const scheduledMessages = await loadSchedulesMessages()
```

Comment:
`createScheduledUserMessage.onPending` return message with incorrect id (`scheduledMessage.scheduledInfo.scheduledMessageId`), therefore, we have to additionally make a request for a list of all scheduled messages, which returns messages with the correct id's